### PR TITLE
Add implementation for ws_value_object_id to string

### PR DESCRIPTION
**Pull request by matthiasbeyer**
Related to #467

### CODE REVIEW

File: src/values/union.c [hunk 1]
```diff
@@ -214,6 +215,18 @@ ws_value_union_tostr(
...
+                size_t strl         = strlen(STR_OF(INTMAX_MAX));
+
+                res = calloc(1, strl + 1);
+                if (!res) {
+                    return NULL;
+                }
+
+                snprintf(res, strl + 3, "id:%ld", id);
+            }
             //!< @todo implement
```
Comment (TheNeikos) [inline src/values/union.c]: You can remove this todo notice then

File: src/values/union.c [hunk 2]
```diff
@@ -214,7 +215,18 @@ ws_value_union_tostr(
...
+                struct ws_object* o = ws_value_object_id_get(&self->object_id);
+                uintmax_t id        = ws_object_uuid(o);
+                size_t strl         = strlen(STR_OF(INTMAX_MAX));
+
+                res = calloc(1, strl + 1);
+                if (!res) {
+                    return NULL;
+                }
+
+                snprintf(res, strl + 3, "id:%ld", id);
```
Comment (neithernut) [inline src/values/union.c]: The buffer behind `res` is too small.

**Comment by neithernut** [comment]
ACK for squashing.

**Comment by matthiasbeyer** [comment]
Squashed btw.

**Comment by neithernut** [comment]
ACK.

**Pull request merged**